data/deduplicator/hash_deactivate_old.go

```diff
@@ -23,7 +23,7 @@ const _HashDeactivateOldDeduplicatorName = "org.tidepool.hash-deactivate-old"
 const _HashDeactivateOldDeduplicatorVersion = "1.1.0"
 
 var _HashDeactivateOldExpectedDeviceManufacturerModels = map[string][]string{
-	"Medtronic": {"523", "723", "551", "751", "554", "754"},
+	"Medtronic": {"523", "723", "551", "751", "554", "754", "1510", "1511", "1512", "1710", "1711", "1712", "1715", "1780"},
```

Comment:
So I believe based on a comment in slack we should be dropping some as we don't have devices to test on prod

> Only support devices we have actually tested with.
> So we drop 620G (1510, 1710) from that list.

Reply:
Yes, that was whether this should be pushed directly to `prd` or not. That's what Eric/Lennart are testing on `stg` now so we can get tested and approved for `prd`.